README.md

```diff
@@ -1,32 +1,32 @@
-# Linuxbrew-fonts
+# Fonts for Homebrew on Linux
 
-Linuxbrew-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-fonts/).
-
-Linuxbrew-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
+homebrew-linux-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts). 
+It is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux since Homebrew Casks are only for macOS.
 
 ## Prerequisites
 
-Fonts are available through homebrew and linuxbrew. So, first you must install [Homebrew](http://brew.sh) or [Linuxbrew](http://linuxbrew.sh). However this repository is **NOT** an "external command” of Homebrew, it is a different from the upstream repository, homebrew/homebrew-cask-fonts. Please caught of that.
-
-**Since Homebrew 1.9 or later supports the both platform, you can use this repository with homebrew in Linux too**
+These fonts are installable with [Homebrew](http://brew.sh).
+This repository is **NOT** an "external command” of Homebrew.
+It is different from the upstream repository, `homebrew/homebrew-cask-fonts`.
+Please be aware of that.
```

Comment:
What does this mean, though? In other words: as a user, this sentence makes me aware of this fact; now what am I supposed to do with this information? Are there things I would have expected to be able to do if I wasn't aware of this? If so, can we offer an example or two to make this concrete?

Reply:
I _think_ the intent is to additionally call out that they're separate and unequal repos: they diverge in content availability and installation mechanism. Maybe this message made more sense when cask was a separate command, which it no longer is.

I'll put up a suggestion with that in mind.

Reply:
https://github.com/Homebrew/homebrew-linux-fonts/pull/33/files#r1120940757